src/services/store.ts

```diff
@@ -214,6 +201,22 @@ export function setDisplayedFields(sceneRef: SceneObject, fields: string[]) {
   localStorage.setItem(`${pluginJson.id}.${PREFIX}.logs.fields`, JSON.stringify(fields));
 }
 
+// Log panel options
```

Comment:
Were these 2 functions just moved?

Reply:
Yeah, might be good to break out the store a bit in the future, I was just organizing a bit